### ISSUE TITLE
SteamAPI and Workshop mods not working 

### ISSUE THREAD

**Issue by M0n7y5** [Rfvgyhn/docker-avorion#11]
Hi, i cant get this container working with workshop mods
i am getting this message 

```
Server version, as listed publicly: 1.2.0.25399
[S_API FAIL] SteamAPI_Init() failed; SteamAPI_IsSteamRunning() failed.
dlopen failed trying to load:
steamclient.so
with error:
steamclient.so: cannot open shared object file: No such file or directory
[S_API FAIL] SteamAPI_Init() failed; unable to locate a running instance of Steam, or a local steamclient.so.
SteamGameServer_Init call failed
Error starting steam-based networking. Falling back to standard TCP/UDP protocols.
The server will not be authenticated via Steam and won't show up in public server lists.
WARNING: The fallback TCP/UDP protocols are deprecated and potentially UNSAFE!
         If you're running a dedicated server, this is HIGHLY discouraged!
         Use steam networking instead; Enable with --use-steam-networking 1
Warning: No RCON password set. RCON disabled.
Found 4 mods in "/home/steam/.avorion/galaxies/avorion_galaxy/modconfig.lua".

```

**Comment by M0n7y5** [comment]
Also this 
![obrazek](https://user-images.githubusercontent.com/17201053/95021204-dc2ceb80-066f-11eb-86ff-f04b12ff0963.png)

**Comment by Rfvgyhn** [comment]
I uploaded a test image. Can you try it and see if your mods work? `rfvgyhn/avorion:1.2.0.25399-dev` If that one works, I'll update the standard tags with the fix.

**Comment by M0n7y5** [comment]
I am gonna try it ASAP. Also i forgot mention about another error. 
`Warning: failed to init SDL thread priority manager: SDL not found`

Which can be fixed by installing SDL2 package.

**Comment by Rfvgyhn** [comment]
I'm going to assume it's fixed. Feel free to reopen if it's not. This fix plus the SDL2 fix is included in the standard tags starting from 1.2.0.25399 and 1.3.0.26854-beta.
